libs/test-common/src/main/java/org/codice/ddf/test/common/configurators/PortFinder.java

```diff
@@ -21,7 +21,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** Class used to find and reserve a range of ports and assign ports within that range. */
+/**
+ * Class used to find and reserve a range of ports, and assign ports within that range by name. For
+ * instance, creating a new instance of {@link PortFinder} will reserve a block of ports, calling
+ * {@link #getPort(String)} or {@link #getPortAsString(String)} the first time will assign a
+ * specific port in that range with the name provided, and calling one of those two methods with
+ * that name will return the same port number afterwards.
+ */
 public class PortFinder implements Closeable {
   private static final Logger LOGGER = LoggerFactory.getLogger(PortFinder.class);
 
```

Comment:
❗️ The `getPort()` method will always increase the nextPort even if it exists in the map. You might want to do something like `return registeredPorts.computeIfAbsent(portKey, k -> nextPort++);`

Reply:
Great catch! Fixed.